### DISCUSSION TITLE
Kubernetes 1.25+ is not compatible with Linkerd 2.11

### DISCUSSION THREAD

**Discussion by drehelis** [linkerd/linkerd2#11048] [General] [answered]
I couldn't find release cycle for Linkerd so I'd assume the attached dropdown menu is the currently maintained and supported versions but I might be wrong here.

![image](https://github.com/linkerd/linkerd2/assets/11921235/c2224783-ad31-43c4-b5ef-96d8df472976)

In 1.25, batch/v1beta1 CronJob API is no longer available. This puts `linkerd-proxy-injector` in a `CrashLoopBackOff ` with the error
```
I0620 08:53:05.809070       1 request.go:1123] Response Body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"the server could not find the requested resource","reason":"NotFound","details":{},"code":404}
E0620 08:53:05.809174       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.21.3/tools/cache/reflector.go:167: Failed to watch *v1beta1.CronJob: failed to list *v1beta1.CronJob: the server could not find the requested resource
```

This should be noted somewhere, with a big noticeable text/color ;-) 

WDYT?

**Comment by alpeb** [comment, marked as answer]
As stated in that doc:

> Stable versions earlier than the previous major version will not receive bugfixes or enhancements.

which means 2.11 is no longer supported.

But thanks for the report, this helps in #11068 figuring out the exact reason of the incompatibility.
